functions/source/soci-index-generator-lambda/Makefile

```diff
@@ -1,5 +1,6 @@
 default:
-	GOOS=linux GOARCH=amd64 go build -o main
+	# Use static builds to make sure we don't have library version issues between the build env and lambda
+	GOOS=linux GOARCH=amd64 go build -tags "osusergo netgo static_build" -ldflags '-extldflags "-static"' -o main
```

Comment:
This seems like a bigger change than just adding region support

Reply:
I was having glibc version issues when testing in China. I think its useful to tie this change to the additional regions